_includes/configuration/text-color.md

```diff
@@ -22,11 +22,13 @@ These settings affect the execution of the `textcolor`. The dimensions and mappi
 
 ### `color_cols`
 
-This option allows specifying the number of columns that can appear on the grid of text colors. The number of rows is calculated based on how many text colors are supplied divided by the specified number of columns.  The `textcolor_cols` and `textcolor_rows` property has been removed in {{site.productname}} 5.
+This option allows specifying the number of columns a the grid of text colors. The number of rows is calculated based on the number of text colors are supplied divided by the specified number of columns. 
 
-**Type:** `String`
+By default, the number of rows and columns is dependent of the number of colors specified using [`color_map`](#color_map), and the dimensions of the grid will be calculated to keep the grid a square. 
```

Comment:
Not sure if you're worried, this isn't quite correct looking at the code. By the looks of it, it'll resize automatically but stop at a minimum 5 columns.